salt/docs/init.sls

```diff
@@ -46,17 +52,40 @@ docsbuild-scripts:
       # Theses are needed to build C extensions.
       - pkg: doc-pkgs
 
-/srv/docsbuild/venv/:
-  virtualenv.managed:
+py36-virtualenv:
+  cmd.run:
     - user: docsbuild
-    - no_deps: True
-    - python: python3
-    - requirements: /srv/docsbuild/scripts/requirements.txt
+    - name: 'python3.6 -m venv --without-pip /srv/docsbuild/venv'
+    - creates: /srv/docsbuild/venv/bin/python
     - require:
-      - git: docsbuild-scripts
-      # Theses are needed to build C extensions.
       - pkg: doc-pkgs
 
+/srv/docsbuild/venv/get-pip.py:
+  file.managed:
+    - user: docsbuild
+    - source: https://bootstrap.pypa.io/get-pip.py
+    - source_hash: sha256=19dae841a150c86e2a09d475b5eb0602861f2a5b7761ec268049a662dbd2bd0c
```

Comment:
this is suboptimal, but I plan to add a checksum to the get-pip release process that we can pull at runtime.